js/layout-javascript/agenda-code.js

```diff
@@ -2401,6 +2409,27 @@ DynamicList.prototype.addDetailViewData = function(entry) {
       content = entry.originalData[dynamicDataObj.column];
     }
 
+    if (dynamicDataObj.type === 'image') {
+      content = entry.originalData[dynamicDataObj.column];
+
+      if (!Array.isArray(content)) {
+        content = content.split(/\n/);
```

Comment:
Why did you make the decision to split the string by linebreaks and not commas, or both?

Reply:
This logic was taken from the previously made [PR](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/425/files#diff-e51aca349bba35deb7224011da8726ab0fd50671ebeaba82184e04766f5b41beR1723) and now when I pasting images URL to the data source they most of the time separated with the new line.
Should we change that and if so what separator would be better to use?

Reply:
Thanks for the reference. Looks like it was the first commit of [this PR](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/425/commits) in Aug 2020. I read through the requirements and there's no such a requirement. Plus, if the value is a number e.g. media ID, then `1.split('\n')` will create an error.

I therefore don't think there's a need to split the value by linebreaks or anything.

Reply:
@tonytlwu When users add images through the form builder we already receiving an array of the image URLs and I didn't see cases when we receive a media ID.
And what about cases when users put image URLs manually in the data source?
In that case, we will receive a string with URLs